CommunityFridgeMapApi/tests/unit/test_fridge_report.py

```diff
@@ -0,0 +1,94 @@
+from multiprocessing import connection
+from CommunityFridgeMapApi.dependencies.python.db import DB_Response
+
+from dependencies.python.db import layer_test
+from dependencies.python.db import get_ddb_connection
+import unittest
+from dependencies.python.db import FridgeReport
+
+class DynamoDbMockPutItem:
+    def __init__(self):
+        pass
+
+    def put_item(self, TableName=None, Item=None, ConditionExpression=None):
+        pass
+
+class FrdgeReportTest(unittest.TestCase):
+
+    def test_required_fields(self):
+        fridge_report = FridgeReport(fridge_report={'fridge_username': 'test', 'status': 'working', 'notes': 'all good out here'}, db_client=None)
+        success, field = fridge_report.has_required_fields()
+        self.assertFalse(success)
+        self.assertEqual(field, "fridge_percentage")
+
+        fridge_report = FridgeReport(fridge_report={'fridge_username': 'test', 'status': 'working', 'notes': 'all good out here', 'fridge_percentage': '33'}, db_client=None)
+        success, field = fridge_report.has_required_fields()
+        self.assertTrue(success)
+
+    def test_set_timestamp(self):
+        fridge_report = FridgeReport(fridge_report={'fridge_username': 'test', 'status': 'working', 'notes': 'all good out here', 'fridge_percentage': '33'}, db_client=None)
+        fridge_report.set_timestamp()
+        self.assertIsNotNone(fridge_report.timestamp)
+    
+    def test_is_valid_status(self):
+        fridge_report = FridgeReport(fridge_report={'status': 'i am a hacker, drop table'}, db_client=None)
```

Comment:
Can we use iteration to write this test case more concisely?

Reply:
Done: https://github.com/CollectiveFocus/CFM_Backend/pull/9/commits/d94f5ce519c57b2d19a22bb36d47cd1f9a228867